src/nstencil_half_multi_3d_tri.cpp

```diff
@@ -81,7 +81,7 @@ void NStencilHalfMulti3dTri::create()
       cutsq = cutcollectionsq[icollection][jcollection];
 
       if (flag_half_multi[icollection][jcollection]) {
-        for (k = 0; k <= sz; k++)
+        for (k = -sz; k <= sz; k++)
```

Comment:
While this works, a more consistent method would be to change the logic in `set_stencil_properties()` such that the "same collection" uses a full stencil instead of changing the definition of a half stencil. I.e.:

```
if (cutcollectionsq[i][i] == cutcollectionsq[j][j]) {
        flag_half_multi[i][j] = false; //true;
        bin_collection_multi[i][j] = i;
}
```

Practically, however, this makes no difference. So I can just make this alternate change in the other pull request.

Reply:
agreed - just make the change in the later PR - thanks for checking